client/components/swipeable/README.md

```diff
@@ -0,0 +1,33 @@
+# Support Info
+
+This component is used to add horizontal swipe controls to a list of elements (pages).
+
+## Example Usage
+
+```js
+import Swipable from 'calypso/components/swipable';
```

Comment:
Could we fix all the typos where we have `swipable` instead of `swipeable`? There are a few more in the PR.